### PR TITLE
update aliases

### PR DESCRIPTION
**Pull request by wyli**
### Description
this PR updates method/class aliases, mainly follows the numpy coding style, for example https://github.com/numpy/numpy/blob/master/numpy/linalg/__init__.py#L73

also adds aliases within modules, following the TF coding style, for example
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/layers/pooling.py#L38

there is a breaking change for `monai.metrics.compute_meandice.compute_meandice`.
in this PR it becomes either `monai.metrics.meandice.compute_meandice` or
`monai.metrics.compute_meandice`.

`export` is still provided as it's useful when the import path is complicated
`alias` is provided for future use (import module by configuration strings)

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or new feature that would cause existing functionality to change)
- [x] Docstrings/Documentation updated

### CODE REVIEW

File: monai/losses/dice.py [hunk 1]
```diff
@@ -202,3 +197,7 @@ def forward(self, pred, ground, smooth=1e-5):
 
         f = (2.0 * intersection * w + smooth) / (denominator * w + smooth)
         return 1.0 - f.mean()  # final reduce_mean across batches and channels
+
+
+dice = Dice = DiceLoss
+generalized_dice = generalised_dice = GeneralizedDiceLoss
```
Comment (Nic-Ma) [inline monai/losses/dice.py]: I think we don't need to consider "s" instead "z" in some spelling, otherwise, there are many code lines related to this...

**Comment by Nic-Ma** [comment]
Hi @wyli ,

As you added many modules to __init__.py, should update all the examples to make the import shorter?
Like the code in [unet_training_dict.py](https://github.com/Project-MONAI/MONAI/blob/master/examples/segmentation_3d/unet_training_dict.py) 
```py
from monai.data.synthetic import create_test_image_3d
from monai.data.utils import list_data_collate
from monai.utils.sliding_window_inference import sliding_window_inference
from monai.metrics.compute_meandice import compute_meandice
from monai.visualize.img2tensorboard import plot_2d_or_3d_image
```

File: monai/losses/dice.py (hunk 1 above)
Comment (wyli) [reply monai/losses/dice.py]: the original paper and niftynet uses s, i’d like to keep this specific alias

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> As you added many modules to **init**.py, should update all the examples to make the import shorter?
> Like the code in [unet_training_dict.py](https://github.com/Project-MONAI/MONAI/blob/master/examples/segmentation_3d/unet_training_dict.py)
> 
> ```python
> from monai.data.synthetic import create_test_image_3d
> from monai.data.utils import list_data_collate
> from monai.utils.sliding_window_inference import sliding_window_inference
> from monai.metrics.compute_meandice import compute_meandice
> from monai.visualize.img2tensorboard import plot_2d_or_3d_image
> ```

thanks, I've updated all the cases, verified the examples

**Pull request merged**